crates/steel-core/src/steel_vm/primitives.rs

```diff
@@ -949,13 +951,248 @@ fn equality_module() -> BuiltInModule {
     module
 }
 
+/// Real numbers ordering module.
+#[steel_derive::define_module(name = "steel/ord")]
 fn ord_module() -> BuiltInModule {
     let mut module = BuiltInModule::new("steel/ord");
+
+    fn steel_ord(left: &SteelVal, right: &SteelVal) -> Result<Option<Ordering>> {
+        Ok(match (left, right) {
```

Comment:
Not sure if you saw this, but this is how `PartialOrd` is implemented for steel values as of now: https://github.com/mattwparas/steel/blob/master/crates/steel-core/src/rvals.rs#L2081

Could we just lift the comparisons for numbers and re use them here?

Reply:
Indeed I didn't see it, thanks

Did I understand correctly that you suggest to improve those existing ones?

Reply:
Sorry for the delay, been a busy few days. I didn't do an exhaustive comparison, but mostly suggesting using the existing comparison code as reference to make sure we're not missing anything - whether that means improving what is there or for the stuff you've written. I'll do another pass shortly to see what the differences are if any

Reply:
Hey, i've reused existing PartialEq since it's there, and updated it as well in order not to loose precision easily, per TODO above